src/panels/config/integrations/ha-config-integration-page.ts

```diff
@@ -550,10 +551,24 @@ class HaConfigIntegrationPage extends SubscribeMixin(LitElement) {
         `ui.panel.config.integrations.config_entry.state.${item.state}`,
       ];
       if (item.reason) {
-        this.hass.loadBackendTranslation("config", item.domain);
-        stateTextExtra = html`${this.hass.localize(
-          `component.${item.domain}.config.error.${item.reason}`
-        ) || item.reason}`;
```

Comment:
The fallback to `|| item.reason` was removed in this PR

Reply:
It probably resolves to an empty string?

Reply:
```
{
    "id": 54,
    "type": "event",
    "event": [
        {
            "type": "updated",
            "entry": {
                "entry_id": "727e03e67745bd15d9923feb667cc0bf",
                "domain": "openweathermap",
                "title": "OpenWeatherMap",
                "source": "user",
                "state": "setup_retry",
                "supports_options": true,
                "supports_remove_device": false,
                "supports_unload": true,
                "supports_reconfigure": false,
                "pref_disable_new_entities": false,
                "pref_disable_polling": false,
                "disabled_by": null,
                "reason": "Please note that using One Call 3.0 requires a separate subscription to the One Call by Call plan. Learn more here https://openweathermap.org/price. If you have a valid subscription to the One Call by Call plan, but still receive this error, then please see https://openweathermap.org/faq#error401 for more info.",
                "error_reason_translation_key": null,
                "error_reason_translation_placeholders": null
            }
        }
    ]
}
```

reason is definitely populated coming back from the websocket api

Reply:
`item.reason` is populated

Reply:
`localize(`component.${item.domain}.config.error.${item.reason}`) || item.reason` fixes the issue

Reply:
https://github.com/home-assistant/frontend/pull/20668